### PR TITLE
Add margins, HackNotts 2019, and Slideshow Karaoke

### PR DESCRIPTION
**Pull request by erinosher**

### CODE REVIEW

**Comment by erinosher** [comment]
I still need to add Slideshow Karaoke

**Comment by erinosher** [comment]
@DanielCordell  can you review ASAP, please :)

**Review by DanielCordell — COMMENTED** [review]
Maybe find a better picture for slideshow karaoke?

File: src/_events/margins-2019/description.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+**About margins**
+
+margins is a free one-day conference for people from marginalised backgrounds that work in/with/around technology (or hope to break into the industry!).
```
Comment (DanielCordell) [inline src/_events/margins-2019/description.md]: margins hyperlink?

File: src/_events/hacknotts-2019/description.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+**HackNotts 2019 has been anounced! It'll be happening on 16th and 17th at the University of Nottingham**
```
Comment (DanielCordell) [inline src/_events/hacknotts-2019/description.md]: Hacknotts hyperlink?

File: src/_events/hacknotts-2019/description.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+## What's a hackathon?
+
+A [hackathon](https://en.wikipedia.org/wiki/Hackathon) is a collaborative programming festival. The main goal is to try to create a project in a relatively short amount of time, such as 24 hours. It's a fantastic opportunity to meet new people, learn from mentors, and make cool stuff.
+
+## What's HackNotts?
+
+HackNotts is our annual hackathon, hosted in the School of Computer Science on Jubilee Campus. This year, the event will be running over the weekend of the 24th and 25th of November, with the opening presentations starting some time around 11am on the Saturday, and the closing presentation will be early in the evening on the Sunday.
```
Comment (DanielCordell) [inline src/_events/hacknotts-2019/description.md]: Don't be afraid of hyperlinks

**Review by DanielCordell — COMMENTED** [review]
Hyperlink boi

**Review by DanielCordell — APPROVED** [review]

**Pull request merged**